modules/base/rendering/renderablemodel.cpp

```diff
@@ -130,18 +130,16 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo ModelScaleInfo = {
         "ModelScale",
         "Model Scale",
-        "This value specifies the scale for the model. If a value for the ModelScale was "
-        "provided in the asset file, you can see and change it here. If instead a unit "
-        "name was provided in the asset, this is the value that that name represents. "
-        "For example 'Centimeter' becomes 0.01. For more information see "
-        "http://wiki.openspaceproject.com/docs/builders/models/model-scale.html.",
+        "The scale of the model. If a numeric value is provided in the asset file, the scale "
+        "will be that exact value. If instead a unit name is provided in the asset, this "
```

Comment:
```suggestion
        "will be that exact value. If instead a unit name is provided, this "
```

The text might not only be shown in relation to an asset and might be confusing in those cases